genienlp/train.py

```diff
@@ -658,6 +658,13 @@ def main(args):
     src_lang = args.train_src_languages.split('+')[0]
     tgt_lang = args.train_tgt_languages.split('+')[0]
 
+    # dump entities if required
+    if args.ned_dump_entity_type_pairs and args.add_entities_to_text == 'append':
+        for task, train_set, val_set in zip(tasks, train_sets, val_sets):
+            ned_dump_entity_type_pairs(train_set, args.data, 'train', task.utterance_field)
+            ned_dump_entity_type_pairs(val_set, args.data, 'eval', task.utterance_field)
+        return
```

Comment:
I would remove the return here as training exiting code with no warnings can cause future confusions wen debugging. Also, dumping can be done before training an actual model when running oracle - we don't wanna run the code twice.